Assets/HoloToolkit/Utilities/Scripts/Singleton.cs

```diff
@@ -83,6 +85,8 @@ protected virtual void Awake()
             else if (!IsInitialized)
             {
                 instance = (T)this;
+                searchForInstance = false;
+                DontDestroyOnLoad(gameObject.GetParentRoot());
```

Comment:
Should this be added in the get finder as well? Otherwise, a Singleton Instance set when get is called doesn't look to be protected this way.

Reply:
Good catch

Reply:
Done